src/components/Chart/AmpereChart.jsx

```diff
@@ -28,6 +35,8 @@ import zoomPanPlugin from './plugins/chart.zoomPan';
 
 import chartCss from './chart.icss.scss';
 
+ChartJS.register(LineElement, PointElement, LinearScale, Title);
```

Comment:
A bit unsure about this one. On one hand, I think it's nice that the component using a graph can register the needed elements. On the other, does it conflict if several files try to register the same set of elements? In that case, I feel it should maybe be put in a more central place like Chart.jsx or something.